asreview/state/sqlstate.py

```diff
@@ -953,99 +912,262 @@ def get_priors(self):
+                    """
+            df = pd.read_sql_query(query, con)
+
+        # Else return all records not yet in the results table.
+        else:
+            query = """SELECT record_table.record_id, results.query_strategy
+                    FROM record_table
+                    LEFT JOIN results
+                    ON record_table.record_id = results.record_id
+                    WHERE results.query_strategy is null
+                    """
+            df = pd.read_sql_query(query, con)
+
+        con.close()
+        return df['record_id']
+
+    def get_labeled(self):
+        """Get the record_ids and labels of the labeled records in order of
+        labeling. If you only want the labeled records, this is more efficient
+        than via 'get_pool_labeled_pending'.
+
+        Returns
+        -------
+        pd.DataFrame
+            Dataframe containing the record_ids and labels of the labeled
+            records, in the order that they were labeled.
+        """
+        con = self._connect_to_sql()
+        query = """SELECT record_id, label FROM results
+         WHERE label is not null"""
+        df = pd.read_sql_query(query, con)
+        con.close()
+        return df
+
+    def get_pending(self):
+        """Get the record_ids of the records pending a labeling decision.
+        If you only want the pending records, this is more efficient
+        than via 'get_pool_labeled_pending'.
+
+        Returns
+        -------
+        pd.Series
+            A series containing the record_ids of the records whose label is
+            pending.
+        """
+        con = self._connect_to_sql()
+        query = """SELECT record_id FROM results WHERE label is null"""
+        df = pd.read_sql_query(query, con)
+        con.close()
+        return df['record_id']
+
+    def get_pool_labeled_pending(self):
+        """Return the labeled and unlabeled records and the records pending a
+        labeling decision.
+
+        Returns
+        -------
+        tuple (pd.Series, pd.DataFrame, pd.Series):
+            Returns a tuple (pool, labeled, pending). Pool is a series
+            containing the unlabeled, not pending record_ids, ordered by the
+            last predicted ranking of the model. Labeled is a dataframe
+            containing the record_ids and labels of the labeled records, in the
+            order that they were labeled. Pending is a series containing the
+            record_ids of the records whose label is pending.
+        """
+        con = self._connect_to_sql()
+
+        query = """SELECT record_table.record_id, results.label,
+                results.rowid AS label_order, results.query_strategy,
+                last_ranking.ranking
+                FROM record_table
+                LEFT JOIN results
+                ON results.record_id=record_table.record_id
+                LEFT JOIN last_ranking
+                ON record_table.record_id=last_ranking.record_id
+                """
```

Comment:
`ORDER BY label_order, ranking` would solve the ordering below, isn't it?

That simplifies 
```
        labeled = df.loc[~df['label'].isna()] \
            .sort_values('label_order') \
            .loc[:, ['record_id', 'label']] \
            .astype(int)
```

```
        labeled = df.loc[~df['label'].isna(), ['record_id', 'label']].astype(int)
```